docs/manual/hacking.rst

```diff
@@ -113,6 +113,16 @@ to ensure your patch complies with reasonable formatting constraints. We also
 request that git commit messages follow the
 `standard format <https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html>`_.
 
+Some common linting errors and how to resolve them:
+
+==================================================== =======================================================
+Lint Error                                           Verbose Diff Command
+==================================================== =======================================================
+``I001 isort found an import in the wrong position`` ``python -m flake8 ./libqtile/ --isort-show-traceback``
+``I005 isort found an unexpected missing import``    ``python -m flake8 ./libqtile/ --isort-show-traceback``
+``BLK100 Black would make changes.``                 ``python -m black --diff ./libqtile/``
```

Comment:
Any reason why you wouldn't just let `isort` and `black` make the changes directly? You can view the changes with `git diff` afterwards.

Reply:
Mainly that I don't know if/how there exists a command for flake8/isort for that so consistency. All ears, though.

Reply:
I just run `black libqtile` or `isort libqtile`...

Reply:
So you would rather see the destructive project-wide auto-fix than the diff for this to be merged? Can change if so

Reply:
Changed.